examples/preference_navigation/lib/home/view/home_page.dart

```diff
@@ -65,12 +65,12 @@ class HomeView extends StatelessWidget {
 class PreferencesList extends StatelessWidget {
   /// {@macro preferences_list}
   const PreferencesList({
-    super.key,
     required this.preferences,
+    super.key,
   });
 
   /// Current preferences stored on device.
-  final Map preferences;
+  final Map<dynamic, dynamic> preferences;
```

Comment:
```suggestion
  final Map<String, dynamic> preferences;
```
should this be `<String, dynamic>` based on the key-value approach?

Reply:
Yes it should.